crates/egui_extras/src/layout.rs

```diff
@@ -232,6 +233,13 @@ impl<'l> StripLayout<'l> {
             child_ui.style_mut().visuals.override_text_color = Some(stroke_color);
         }
 
+        if flags.overline {
+            child_ui.painter().line_segment(
+                [max_rect.left_top(), max_rect.right_top()],
```

Comment:
nit: can be slightly simpler

```suggestion
            child_ui.painter().hline(
                max_rect.x_range(),
                max_rect.top(),
```